src/node.rs

```diff
@@ -453,6 +456,30 @@ impl MinerStats {
     }
 }
 
+#[derive(StructOpt, Debug)]
+pub struct MinerRevert {
+    #[structopt()]
+    revert: u32,
+}
+
+impl MinerRevert {
+    pub fn parse(self) -> Result<String> {
+        // Initialize the node.
+        let node = Node::from_iter(&["snarkos", "--norpc", "--verbosity", "0"]);
+
+        let ip = "0.0.0.0:1000".parse().unwrap();
```

Comment:
since there is no reason to connect, the IP should be localhost; as for the port, it should be the snarkOS default port number (or it should be configurable), otherwise the revert won't work

Reply:
Yep, I agree with you. The port will have a default value, or get through the command. what's your idea? If you agree with me, I will push the changes. By the way, as I followed the above code style, the changes will break the legacy style.